src/main/java/net/sf/jabref/gui/event/Event.java

```diff
@@ -0,0 +1,21 @@
+package net.sf.jabref.gui.event;
+
+/**
+ * Represents a medium which can be used
+ * to pass objects on event occurrence.
+ */
+
+public class Event {
```

Comment:
I agree with @koppor  From the api doc the EventObject looks like that is's main purpose is to be used in GUI swing/awt events.